superset/assets/javascripts/explorev2/components/controls/Filter.jsx

```diff
@@ -4,6 +4,9 @@ import Select from 'react-select';
 import { Button, Row, Col } from 'react-bootstrap';
 import SelectControl from './SelectControl';
 
+const ArrayFilterOps = ['in', 'not in'];
```

Comment:
var names should be lower case,`arrayFilterOps`, strFilterOps. or since these are constants `ARRAY_FILTER_OPTIONS`